.github/workflows/typescript.yml

```diff
@@ -0,0 +1,66 @@
+name: Typescript
+
+on:
+  pull_request:
+    branches:
+    - main
+    - release-*
+    - feature/*
+  push:
+    branches:
+    - main
+    - release-*
+    - feature/*
+
+jobs:
+  check-typescript-changes:
+    name: Check whether tests need to be run based on diff
+    runs-on: [ubuntu-latest]
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - uses: antrea-io/has-changes@v2
+      id: check_diff
+      with:
+        paths: plugins/grafana-custom-plugins/*
+    outputs:
+      has_changes: ${{ steps.check_diff.outputs.has_changes }}
+
+  test-unit:
+    needs: check-typescript-changes
+    if: ${{ needs.check-typescript-changes.outputs.has_changes == 'yes' }}
+    name: Unit test
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [16.x]
```

Comment:
We usually don't specify node-version. Do we need to pin it for some reason?

Reply:
Yes, you're right. We don't need to specify the node version. Thanks.